nservicebus/azure-storage-queues/native-delayed-delivery.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Native delayed delivery
+component: ASQ
+versions: '[7.1,]'
+tags:
+- Azure
+- Cloud
+- Delayed delivery
+- ASQ
+- Azure Storage Queues
+reviewed: 2016-10-21
+---
+
+[Delayed delivery](/nservicebus/messaging/delayed-delivery) enables sending messages that are delivered at a later time. 
```

Comment:
I will omit the `feature` word though. It has a strong connotation from the core, and I don't want to carve a new meaning for it.